cds-snc/c19-benefits-node
# Add a script to validate the installation was successful

## Description

Adds a script in `./bin/` that checks the latest SHA on the staging branch and compares it with the SHA injected into the container and that is exposed as a github tag. 

It does this by running a curl to download the first page of the app and then using some ugly grep/awk to extract the sha.

## To Test

run the following command 

`./bin/verifyinstall.sh`

You should see some output like the following: 

```
Verifying Development Deployment
latest commit in Development f647efa4d5c2283e3c31cbfbdf8b54ff749e58bf
🛑 https://cv19benefits-appservice-dev.azurewebsites.net/en/start sha's don't match
f647efa4d5c2283e3c31cbfbdf8b54ff749e58bf !== f647efa4d5c2283e3c31cbfbdf8b54ff749e58bf

Verifying Production Deployment
latest commit in Staging fc25c9c62e03af25428bf3971df9a62dbfc509b9
✅ https://covid-benefits.alpha.canada.ca/en/start verified
✅ https://covid-prestations.alpha.canada.ca/fr/debut verified
```
I've confirmed this works on MacOs and it should work on Linux as well.
